apps/condo/domains/acquiring/schema/RegisterMultiPaymentService.js

```diff
@@ -205,7 +366,7 @@ const RegisterMultiPaymentService = new GQLCustomSchema('RegisterMultiPaymentSer
         },
         {
             access: true,
-            type: 'input RegisterMultiPaymentServiceConsumerInput { consumerId: String!, receipts: [RegisterMultiPaymentReceiptInfoInput!]! }',
+            type: 'input RegisterMultiPaymentServiceConsumerInput { consumerId: String @deprecated(reason: "consumerId is deprecated, use serviceConsumer instead."), """consumerId is deprecated, use serviceConsumer instead. Required field (in future)""" serviceConsumer: ServiceConsumerWhereUniqueInput, receipts: [RegisterMultiPaymentReceiptInfoInput!] @deprecated(reason: "receipts is deprecated, use billingReceipts instead."), """receipts is deprecated, use billingReceipts instead. Required field (in future)""" billingReceipts: [BillingReceiptWhereUniqueInput!] }',
```

Comment:
probably its better to leave `receipts` but change their type it it's really backward compatible with mobile clients

Reply:
then I can remove RegisterMultiPaymentReceiptInfoInput